arq/worker.py

```diff
@@ -475,6 +501,12 @@ async def job_failed(exc: Exception) -> None:
                 if e.defer_score:
                     incr_score = e.defer_score + (timestamp_ms() - score)
                 self.jobs_retried += 1
+            elif job_id in self._aborting_tasks and isinstance(e, asyncio.CancelledError):
+                logger.info('%6.2fs 🛇  %s aborted', t, ref)
```

Comment:
The symbol here doesn't work for me on mac (I can't see it in the browser in github, in pycharm or in the terminal)

How about using the "CIRCLED DIVISION SLASH" symbol ⊘?

That works nicely everywhere and [apparently](https://en.wikipedia.org/wiki/No_symbol#Unicode_Combining_Character) has good font support.

![image](https://user-images.githubusercontent.com/4039449/115602188-8d2a6900-a2d6-11eb-858c-6c35829ec8e4.png)